packages/dashboard/src/components/appbar.tsx

```diff
@@ -18,13 +19,14 @@ export interface AppBarProps {
   toggleShowOmniPanel(): void;
   showSettings(show: boolean): void;
   showHelp(show: boolean): void;
+  tooltips?: boolean;
```

Comment:
What do you think about setting this tooltip as a context instead of a prop?

Reply:
I could give this a shot, the implementation does bother me too ((':